README.md

```diff
@@ -1,135 +1,221 @@
+
+```javascript
+  const email = 'johnnybravo@gmail.com' // Customer's email address
+
+  window.metrilo.identify(email)
+```
+
+#### viewPage
+
+Customer has visited a page on your website. It can be the homepage, or any other page.
+
+```javascript
+  const url = 'http://myshop.com/amazing-page' // Absolute URL to the visited page
+  const params = {
+    name: 'My Amazing page' // The name of the page in the shop
+  }
+
+  window.metrilo.viewPage(url, params)
+```
+
+#### viewArticle
+
+Customer has visited an article page.
+
+```javascript
+  const articleId = '42' // Unique ID of article resource in the shop's database
+  const params = {
+    name: 'How to increase sales' // Article display name in the shop
+    url: 'http://myshop.com/how-to-increase-sales' // Landing page for the article resource
+  }
+
+  window.metrilo.viewArticle(articleId, params)
+```
+
+#### viewCategory
+Customer has visited a category page.
+
+```javascript
+  const categoryId = '9912' // Unique ID of category resource in the shop's database
+
+  window.metrilo.viewCategory(categoryId)
+```
+
+#### viewProduct
+Customer has visited a product page.
+
+```javascript
+  const productId = '312' // The ID of the product in the shop's database
+
+  window.metrilo.viewProduct(productId)
+```
+
+#### search
+
+Customer has searched in the shop (this supports only a plain query string without further filtering).
+
+```javascript
+  const query = 'a plant' // Search query string that the visitor entered in the search form
+  const url = 'http://myshop.com/search/?s=a+plant' // The URL that the visitor landed on when submitting the search form
+
+  window.metrilo.search(query, url)
+```
+
+#### addToCart
+Customer has added a product to the cart.
+
+```javascript
+  const productId = '312' // The ID of the added product
+  const quantity = 2 // The quantity of added items
+
+  window.metrilo.addToCart(productId, quantity)
+```
+
+#### removeFromCart
+Customer has removed a product from the cart.
+
+```javascript
+  const productId = '312' // The ID of the removed product
+  const quantity = 1 // The quantity of removed items
+
+  window.metrilo.removeFromCart(productId, quantity)
+```
+
+#### checkout
+Customer has started the checkout process.
+
+```javascript
+  window.metrilo.checkout()
+```
+
+#### applyTags
+Adds tags to a customer.
+
+```javascript
+  const tags = ['regular', 'big_spender'] // Tag labels that are assigned to the customer in the Metrilo application
+
+  window.metrilo.applyTags(tags)
+```
+
+#### customEvent
+Sends a custom event (anything you want to track).
+
+```javascript
+  const customEventName = 'paypal_payment' // The name of the custom event
+
+  window.metrilo.customEvent(customEventName)
+```
+
+## Troubleshooting
+
+### Error codes
```

Comment:
I think we should include a link to the error codes somewhere that is more obvious. If I am looking to find them, I must know that they are here at the bottom. I think we should include a link at the top as a final step to the others (e.g. 5. Troubleshooting)

Reply:
There is a [separate ticket](https://metrilojira.atlassian.net/browse/KYP-1075) for error codes and troubleshooting - I'm just leaving it the way it is now - that ticket is going to define where we're going to put the links to the troubleshooting section